packages/flutter/lib/src/material/data_table.dart

```diff
@@ -495,8 +501,20 @@ class DataTable extends StatelessWidget {
   /// If null, [DataTableThemeData.dataRowHeight] is used. This value defaults
   /// to [kMinInteractiveDimension] to adhere to the Material Design
   /// specifications.
+  @Deprecated(
+    'Use dataRowHeightSettings instead. '
+    'This feature was deprecated after v2.13.0-0.4.pre.',
+  )
   final double? dataRowHeight;
 
+  /// {@template flutter.material.dataTable.dataRowHeightSettings}
+  /// The height settings for [DataRow]s.
```

Comment:
maybe:

The height settings of each row excluding the row that contains column headings.